shambomon/assets/js/components/messages.jsx

```diff
@@ -6,15 +6,15 @@ function getMessages(messages) {
   if (messages.length > 0) {
     // Make the newest set of messages a different color
     for (var i = messages.length - 1; i >= messages.length - 3; i--) {
-      msgs.push(<p className="last-messages">>> {messages[i]}</p>);
+      msgs.push(<p className="last-messages" key={i}> {messages[i]}</p>);
```

Comment:
I think you accidentally got rid of the ">>" before each message.

Reply:
Oops I thought that was a typo lol

Reply:
Lol it's okay.